src/models/event.js

```diff
@@ -299,7 +310,16 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
-     * Attempt to decrypt this event.
+     * Check if this event is currently being decrypted.
+     *
+     * @return {boolean} True if this event is currently being decrypted, else false.
+     */
+    isBeingDecrypted: function() {
+        return this._decrypting;
+    },
+
+    /**
+     * Start the process of trying to decrypt this event.
```

Comment:
Feels icky API-wise if we don't return a promise even if this is an internal function. I would instinctually be asking myself "**why** aren't we returning a promise here" if I were to hack on this code, thinking there may be a deeper meaning other than "we have never cared so we've never bothered". If you can make it return a promise and leave it up to the callers to use/discard that'd be appreciated.